x/concentrated-liquidity/lp_test.go

```diff
@@ -981,7 +981,7 @@ func (s *KeeperTestSuite) TestAddToPosition() {
 				amount0Minimum: sdk.NewInt(1997960),
 				expectedError: types.InsufficientLiquidityCreatedError{
 					Actual:      sdk.NewInt(1997954),
-					Minimum:     sdk.NewInt(1997960),
+					Minimum:     sdk.NewInt(2996936),
```

Comment:
Can you please define a success test case where if the minimum was one more unit, the test would fail? Also, please explain how the values are computed for that test cases.